back/spec/acceptance/areas_spec.rb

```diff
@@ -24,6 +24,13 @@
       expect(json_response[:data].size).to eq 5
     end
 
+    example 'List all topics for onboarding' do
+      @areas.last.update!(include_in_onboarding: true)
```

Comment:
I think I'm OK with it as it is, but not sure why the request param would be called `for_onboarding` instead of `include_in_onboarding` like it is in the model? Just feels like a bit of a mismatch to me